Dockerfile

```diff
@@ -0,0 +1,8 @@
+# Dockerfile to build the application on Render
+FROM maven:3.8.5-openjdk-17
+COPY /src /app/src
+COPY pom.xml /app/
+WORKDIR /app
+# Disable tests on Render
+RUN mvn clean package -DskipTests
```

Comment:
If we are skipping tests, then it would be best to use github actions to run them on the repository. Could you add a workflow for this?